WF-73 Added tests for new workflow process-instance STOP endpoint

FaraiMutero opened the pull request.

FaraiMutero committed:
WF-73 Added tests for new workflow process-instance STOP endpoint

taylordowns2000 reviewed (commented):
Looks great but because the Circle setup isn't working for the `mock` I can't actually tell if these tests are passing!

Does it run properly on your local machine @FaraiMutero ?

FaraiMutero committed:
WF  Fixed missed instanceId parameter in /instances/{instanceId}/stop endpoint

FaraiMutero committed:
Merge branch 'main' of https://github.com/GovStackWorkingGroup/bb-workflow into add-stop-process-instance

FaraiMutero commented:
Hey @taylordowns2000. I have added the stop-process-instance endpoint and fixed a related bug in swagger.yml. Refering to our previous chat, I'm satisfied that there is solid case to provide a way to stop a process instance. Here is a similar implementation for [Camunda](https://docs.camunda.org/manual/7.7/webapps/cockpit/bpmn/process-instance-view/?__hstc=218867270.30f498c593fbfa91a6f3a688fe6c6468.1697720024704.1697720024704.1697720024704.1&__hssc=218867270.2.1697720024705&__hsfp=224056415&_gl=1*24dhrg*_ga*MjAxNjQ2OTgxNS4xNjk3NzIwMDIy*_ga_4EYN8X5FNR*MTY5NzcyMDAyMS4xLjEuMTY5NzcyMTUwMy4yMS4wLjA.#cancel-a-process-instance).
